docs/source/user-guide/vision/supported_tasks_and_formats.rst

```diff
@@ -0,0 +1,124 @@
+.. _supported_tasks_and_formats:
+
+===========================
+Supported Tasks and Formats
+===========================
+
+In order to accurately analyze the images, labels and predictions provided in the
+:doc:`VisionData object </user-guide/vision/vision_data_class>`,
+they must be provided in deepchecks accepted format. In this section we will describe the supported formats
+for each supported task type.
+
+The data formats are validated at the creation of the VisionData object based on the first batch of data. However,
+in addition to the automatic validations, it is also important to make sure visually that the data was provided in the
```

Comment:
```suggestion
you can always make sure the data was provided in the correct format by visualizing it, using 
```

Reply:
Now I'm sad :(

Honestly I think it was better - no need to repeat the automatic validations now (because of the previous change in the sentence). And moreover, I think it's better to say that "you can if you want" rather than "it's important to" as it's not necessarily important to.